src/GafferImageUI/ImageGadget.cpp

```diff
@@ -99,9 +99,7 @@ void findUsableTextureFormats( GLenum &monochromeFormat, GLenum &colorFormat )
 		else
 		{
 			IECore::msg( IECore::Msg::Warning, "ImageGadget",
-				"Could not find supported floating point texture format in OpenGL.  GPU image"
-				" viewer path will be low quality, recommend switching to CPU display transform,"
-				" or resolving graphics driver issue."
```

Comment:
Maybe we should remove the warning from here entirely (because ImageGadget itself isn't reliant on the precision) and put a different warning in ViewportGadget's `checkGLArbTextureFloat()`? Although I don't think really the warning text matters all that much - if things are as bad as you say, then that will first be obvious visually in the viewer, and the warning's purpose is just to say why that happened.

Reply:
As discussed offline, I've moved the warning in https://github.com/GafferHQ/gaffer/pull/5211/commits/efc906165b52bbb13691863b0d22da37dbf33725. This did require more rejigging than anticipated, to avoid the warning triggering test failures on platforms without floating point textures. Worth taking a decent look at that commit.

Reply:
LGTM